bin/build

```diff
@@ -13,6 +13,6 @@ BUILD_SCRIPT="cd /src && "\
 "rm -rf build && mkdir build && "\
 "go build -o build/codeclimate-govet ."
 
-docker run -v "$SRC_ROOT:/src" yunspace/alpine-golang /bin/sh -c "$BUILD_SCRIPT"
+docker run -v "$SRC_ROOT:/src" golang:latest /bin/sh -c "$BUILD_SCRIPT"
```

Comment:
hey thanks for comment! sorry I missed this before. Commented in the govet PR: my initial reason was because the golang:alpine image doesn't have git installed - which we use to pull in the cc-engine-go dependency in the build script.

Any other idea for a workaround there?